front/app/components/IdeaCards/IdeasWithFiltersSidebar/index.tsx

```diff
@@ -122,9 +122,12 @@ const ContentRight = styled.div<{ filterColumnWidth: number }>`
   justify-content: flex-start;
   align-self: flex-start;
   margin-left: ${gapWidth}px;
-  position: relative;
+  max-height: calc(100vh - 120px);
   position: sticky;
   top: 100px;
+  overflow-y: scroll;
+  padding-left: 8px;
+  padding-right: 8px;
 `;
```

Comment:
New padding adds some extra space between the filter boxes and the scrollbar, which I think looks nicer.